src/search.cpp

```diff
@@ -68,7 +68,8 @@ namespace {
   const int skipPhase[] = { 0, 1, 0, 1, 2, 3, 0, 1, 2, 3, 4, 5, 0, 1, 2, 3, 4, 5, 6, 7 };
 
   // Razoring and futility margin based on depth
-  const int razor_margin[4] = { 483, 570, 603, 554 };
+  // razor_margin[0] is unused as long as depth >= ONE_PLY in search
+  const int razor_margin[4] = { 0, 570, 603, 554 };
```

Comment:
@mcostalba :  if you don't like redundant array sizes, take a look at pawn.cpp and evaluate.cpp